zipkin-example/src/main/java/com/example/helloworld/HelloWorldApplication.java

```diff
@@ -19,7 +21,7 @@ public static void main(String[] args) throws Exception {
 
     @Override
     public String getName() {
-        return "hello-world";
+        return "dropwizard-zipkin";
```

Comment:
I left this `hello-world` as it was building upon http://www.dropwizard.io/0.9.3/docs/getting-started.html